### PR TITLE
remove welcome area from app

### PR DESCRIPTION
**Pull request by sqs**
The about site at https://about.sourcegraph.com replaces the in-app welcome area. We may re-add it later, but it is not used right now.

### CODE REVIEW

File: web/src/nav/NavLinks.tsx [hunk 1]
```diff
@@ -104,12 +88,12 @@ export class NavLinks extends React.PureComponent<Props> {
                                 </a>
                             </li>
                         )}
-                        {this.props.location.pathname !== '/welcome' && (
-                            <li className="nav-item">
-                                <Link to="/help" className="nav-link">
-                                    Help
-                                </Link>
-                            </li>
+                        (
```
Comment (felixfbecker) [inline web/src/nav/NavLinks.tsx]: This looks like a JSX artifact (also see Snapshot)
```suggestion
```

File: web/src/nav/NavLinks.tsx [hunk 2]
```diff
@@ -104,12 +88,12 @@ export class NavLinks extends React.PureComponent<Props> {
...
+                        <li className="nav-item">
+                            <Link to="/help" className="nav-link">
+                                Help
+                            </Link>
+                        </li>
                         )}
```
Comment (felixfbecker) [inline web/src/nav/NavLinks.tsx]:
```suggestion
```

File: web/src/nav/__snapshots__/NavLinks.test.tsx.snap [hunk 3]
```diff
@@ -49,58 +38,49 @@ Array [
...
-
 exports[`NavLinks unauthed Sourcegraph.com /foo 1`] = `
 Array [
   "Search /search",
-  "Welcome /welcome",
   "WebActionsNavItems",
-  "Explore /explore",
+  "(",
   "Extensions /extensions",
+  ")}",
```
Comment (felixfbecker) [inline web/src/nav/__snapshots__/NavLinks.test.tsx.snap]: JSX artifacts (more in this file)

**Comment by sqs** [comment]
fixed thx!

File: web/src/nav/__snapshots__/NavLinks.test.tsx.snap (hunk 3 above)
Comment (felixfbecker) [reply web/src/nav/__snapshots__/NavLinks.test.tsx.snap]: Still in here?

Comment (sqs) [reply web/src/nav/__snapshots__/NavLinks.test.tsx.snap]: oops, fixed

**Pull request merged**